src/openseadragon.js

```diff
@@ -2294,7 +2294,25 @@ function OpenSeadragon( options ){
 
                 request.send(null);
             } catch (e) {
-                $.console.log( "%s while making AJAX request: %s", e.name, e.message );
+                var msg = e.message;
```

Comment:
I believe all of this IE code has been removed on master, so I think it's being mistakenly added back here (due to a merge conflict). Would you agree, @msalsbery ?

I think this whole section simply needs to become :

```
$.console.error( "%s while making AJAX request: %s", e.name, e.message );
```

@athammana can you make that change?

Reply:
@iangilman  Yeah weird it doesn't show conflict, but master should definitely be merged into  athammana:master ...there's code right below there that's now gone